packages/theme/src/scale-medium.css

```diff
@@ -11,7 +11,3 @@ governing permissions and limitations under the License.
 */
 
 @import '@spectrum-web-components/styles/scale-medium.css';
-
```

Comment:
This looks like extra code. Particularly, when using the `sp-theme` element, all of these sheets go after https://github.com/adobe/spectrum-web-components/blob/master/packages/theme/src/theme.css#L16 which already has this rule. I can be convinced that I'm being overly worried here.